content/en/blog/2020/new-deployment-model/index.md

```diff
@@ -0,0 +1,103 @@
+---
+title: New Deployment Models for Istio
```

Comment:
This title seems pretty confusing. It's really just a variation of the shared control plane multicluster model, where the primary cluster has no user services running, just the control plane.
```suggestion
title:  An Isolated Control Plane Deployment Model
```

Reply:
It is actually not just a variation of the shared control plane multicluster model.  This is a single cluster model but istiod runs outside of the cluster for easier management by another team/provider.

Reply:
Another title suggestion would be "Introduce Central Istiod where Istiod is managed separately"

Reply:
Another suggestion "A Deployment Model for Running the Istio Control Plane Outside of Your Workload Cluster"

Reply:
@rvennam That sounds more like a description than a title :) But it is getting to the point. All of the "workloads" are running in what Istio terminology calls [remote clusters](https://preliminary.istio.io/latest/docs/reference/glossary/#remote-cluster). The control plane is running somewhere else. I'm assuming that in such a configuration, istiod wouldn't need to be running in a kubernetes cluster, it just happens to be in this example.

I think that this article needs to align with Istio terminology (e.g., use, and reference, the glossary terms). I also think that we need to describe this model in the current docs by updating [this section of the deployment guide](https://preliminary.istio.io/latest/docs/ops/deployment/deployment-models/#control-plane-models). It should include a picture and description of this "all remote clusters" model. Then the blog can be presented as an example of this model, one where the control plane is running in another kubernetes cluster as opposed to in one of the mesh/workload clusters (aka, a primary cluster).

Reply:
To be clear, `primary cluster` does not imply anything about where user workloads run. It simply implies that the control plane runs in *that* cluster.